### PR TITLE
allows users to add their own markers to the map

### PR DESCRIPTION
**Pull request by sundyx**
Utilized the maps API again to add the ability for users to create points on a map. Instead of changing the previous map, I've gone ahead and added a new HTML file called user-map so it would be less confusing to distinguish between the boba locations and the locations from the user input. Not sure why all the maven files were added though...

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -139,30 +139,62 @@ public void readMessage(Entity entity, List<Message> messages, String user) {
...
   /** Stores the User in Datastore. */
- public void storeUser(User user) {
-  Entity userEntity = new Entity("User", user.getEmail());
-  userEntity.setProperty("email", user.getEmail());
-  userEntity.setProperty("aboutMe", user.getAboutMe());
-  datastore.put(userEntity);
- }
+  public void storeUser(User user) {
+    Entity userEntity = new Entity("User", user.getEmail());
+    userEntity.setProperty("email", user.getEmail());
```
Comment (sabingaire) [inline src/main/java/com/google/codeu/data/Datastore.java]: From line 142 to 147, there is an extra space at the start of the line.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -139,30 +139,62 @@ public void readMessage(Entity entity, List<Message> messages, String user) {
...
   * Returns the User owned by the email address, or
   * null if no matching User was found.
   */
- public User getUser(String email) {
+  public User getUser(String email) {
 
-  Query query = new Query("User")
+   Query query = new Query("User")
     .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
-  PreparedQuery results = datastore.prepare(query);
```
Comment (sabingaire) [inline src/main/java/com/google/codeu/data/Datastore.java]: There is an extra space to the start of the line in 153,155,157,158,159

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 3]
```diff
@@ -139,30 +139,62 @@ public void readMessage(Entity entity, List<Message> messages, String user) {
...
+      }
+      return markers;
+   }
 
-  String aboutMe = (String) userEntity.getProperty("aboutMe");
-  User user = new User(email, aboutMe);
 
-  return user;
- }
+   public void storeMarker(UserMarker marker) {
```
Comment (sabingaire) [inline src/main/java/com/google/codeu/data/Datastore.java]: Blank line added. I think blank line 192 is unnecessary. Is it true @jonathantanmy ?

File: pom.xml [hunk 4]
```diff
@@ -57,8 +57,8 @@ limitations under the License.
 
     <dependency>
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-datastore</artifactId>
-      <version>1.52.0</version>
+      <artifactId>google-cloud-translate</artifactId>
```
Comment (jonathantanmy) [inline pom.xml]: What is the reason for changing this part?

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (jonathantanmy) [reply src/main/java/com/google/codeu/data/Datastore.java]: Yes - this is an example of unnecessary indentation. Please restore it to what it was previously. (The easiest way is probably to download the master branch from GitHub as a ZIP and copy over the existing lines.)

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 3 above)
Comment (jonathantanmy) [reply src/main/java/com/google/codeu/data/Datastore.java]: Line 192 wasn't added - it's white, which means that it's existing, so don't worry about this, Sandy.

**Review by sabingaire — APPROVED** [review]

**Pull request merged**